plans/PROGRESS.md

```
@@ -31,7 +31,92 @@ each completed plan gate.
+run (see `run()`'s own `TRUNCATE core.play, core.pitch, ...` before
+`_build_pitches` is called) -- the very next scheduled conform run
+populates the new columns for the full history automatically, the same
+"truncate everything, rebuild from raw" design this table already has.
+
+**Proposed spray-angle formula (a public, well-established convention --
+independently reimplemented here, not copied from any specific library's
+code, consistent with this project's `wSB`/`wOBA`/`wRC+` precedent of
+reimplementing public formulas rather than scraping or vendoring code):**
+```
+spray_angle_deg = atan2(hc_x - 125.42, 198.27 - hc_y) * 180 / pi()
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)